sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -151,11 +222,70 @@ if [[ "$RUNNER" == "flink" || "$RUNNER" == "spark" || "$RUNNER" == "portable" ]]
     java -jar $EXPANSION_SERVICE_JAR $EXPANSION_PORT &
     EXPANSION_PID=$!
   fi
+fi
+
+if [[ "$RUNNER" == "dataflow" ]]; then
+  # Verify docker and gcloud commands exist
+  command -v docker
+  docker -v
+  command -v gcloud
+  gcloud --version
+
+  # ensure gcloud is version 186 or above
+  TMPDIR=$(mktemp -d)
+  gcloud_ver=$(gcloud -v | head -1 | awk '{print $4}')
+  if [[ "$gcloud_ver" < "186" ]]
+  then
+    pushd $TMPDIR
+    curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-186.0.0-linux-x86_64.tar.gz --output gcloud.tar.gz
+    tar xf gcloud.tar.gz
+    ./google-cloud-sdk/install.sh --quiet
+    . ./google-cloud-sdk/path.bash.inc
+    popd
+    gcloud components update --quiet || echo 'gcloud components update failed'
+    gcloud -v
+  fi
 
-  ARGS="$ARGS --endpoint=$ENDPOINT --expansion_addr=$EXPANSION_ADDR"
+  # Build the container
+  TAG=$(date +%Y%m%d-%H%M%S)
+  CONTAINER=us.gcr.io/$PROJECT/$USER/beam_go_sdk
```

Comment:
The $USER here is just the standard OS $USER executing the script then? Just confirming something wasn't missed.

Reply:
Hmm, looks like it. When I copied the code over I didn't notice that no USER variable was defined. On the other hand, this is just used to generate a directory for the script to store containers, so it's probably unnecessary to have a flag for this unless it causes problems.